src/otx/recipe/semantic_segmentation/dino_v2_tile.yaml

```diff
@@ -0,0 +1,59 @@
+model:
+  class_path: otx.algo.segmentation.dino_v2_seg.DinoV2Seg
+  init_args:
+    label_info: 2
+    model_name: dinov2_vits14
+    input_size:
+      - 560
+      - 560
```

Comment:
```suggestion
```
Could you remove it? data.input_size is automatically passed as a model argument.